scripts/pi-hole/js/index.js

```diff
@@ -295,31 +295,30 @@ function updateTopClientsTable(blocked) {
     // Add note if there are no results (e.g. privacy mode enabled)
     if (jQuery.isEmptyObject(data.clients)) {
       clienttable.append('<tr><td colspan="3"><center>- No data -</center></td></tr>');
-    }
-
-    // Populate table with content
-    data.clients.forEach(function (client) {
-      // Sanitize client
-      let clientname = client.name;
-      if (clientname.length === 0) clientname = client.ip;
-      url =
-        '<a href="queries.lp?client_ip=' +
-        encodeURIComponent(client.ip) +
-        '">' +
-        utils.escapeHtml(clientname) +
-        "</a>";
-      percentage = (client.count / sum) * 100;
-
-      // Add row to table
-      clienttable.append(
-        "<tr> " +
+    } else {
```

Comment:
Wouldn't it be easier to simply `return` here instead of making so many changes by moving stuff into a new `else` branch?